demos/python_demos/speech_recognition_demo/utils/default_alphabet_example.conf

```diff
@@ -1,8 +1,7 @@
-# Each line in this file represents the Unicode codepoint (UTF-8 encoded)
-# associated with a numeric label.
-# A line that starts with # is a comment. You can escape it with \# or \\ if you wish
-# to use '#' or '\' as a label.
-
+# Each line in this file represents the Unicode codepoint (UTF-8 encoded) associated with a numeric label.
+# A line that starts with # is a comment. You can escape it with \# or \\ if you wish to use '#' or '\' as a label.
+# The following line in this example contains a single whitespace character, which is the first symbol of the alphabet:
+ 
```

Comment:
I'm not a fan of formats that require invisible whitespace. Maybe you could change the demo to use `shlex.split` to parse this file? That would be compatible with the current syntax, and you could express this character as `' '`.

Reply:
I'd like to keep backward compatibility with Mozilla's ctcdecode alphabet format. So I changed to accept both whitespace and \w, and put \w into the example file.